app/src/components/Projects.svelte

```diff
@@ -1,26 +1,31 @@
 <script>
-  import { Text, Title, Icon, Numeric, Caption } from "../DesignSystem";
+  import { Text, Title, Icon, Numeric, Caption, Button } from "../DesignSystem";
   import ProjectCard from "./ProjectCard.svelte";
   import { projectOverview } from "../path.js";
   import { gql } from "apollo-boost";
   import { getClient, query } from "svelte-apollo";
-  import { link } from "svelte-spa-router";
+  import { link, push } from "svelte-spa-router";
 
   const GET_PROJECTS = gql`
     query Query {
       projects {
-        id {
+        id
+        metadata {
+          description
           name
-          domain
+          imgUrl
         }
-        description
-        name
-        imgUrl
       }
     }
   `;
 
-  const projects = query(getClient(), { query: GET_PROJECTS });
+  const client = getClient();
+  const projects = query(client, { query: GET_PROJECTS });
+  projects.refetch();
+
+  const createProject = () => {
+    push("/projects/new");
```

Comment:
Why do we use push directly, instead of an href? Also can centralise routing strings?

Reply:
Because it is an async action via a button push. How would you do it with a href?

Reply:
+1 for routing strings.